### ISSUE TITLE
Trying to add new occupancy type abbrevation fails

### ISSUE THREAD

**Issue by yuskey** [GreenBuildingRegistry/usaddress-scourgify#20]
Hey guys,

So I created an address_constants.yaml file with the following:

```
insertion_method: update
KNOWN_ODDITIES:
    'developed by HOST': ''
    '#': '#'
OCCUPANCY_TYPE_ABBREVIATIONS:
    '#': '#'
```

And within my code I do the following:
```
import os
from pathlib import Path
os.environ['ADDRESS_CONFIG_DIR'] = str(Path().absolute())
print(os.getenv('ADDRESS_CONFIG_DIR'))
```
which creates an enviroment variable called ADDRESS_CONFIG_DIR that points to the current directory, which is also where I have the address_constants.yaml file. Lets use the following example, the file is located at /home/tmp/address_constants.yaml
then the ADDRESS_CONFIG_DIR environment variable would be /home/tmp. 

I then run the following:
```
from scourgify import normalize_address_record

addr = '1234 Example lane # 12, City, CA, 12345'
out_dict = normalize_address_record(addr)
```

It doesnt seem to split 1234 Example Ln and # 12 as two different address lines. 

I believe I followed the correct steps to add a new occupancy type abbreviation to split on but I could be wrong, has anyone run into something similar?

**Comment by fablet** [comment]
The problem seems to be with the tagging from usaddress.  

You can do a quick check of how usaddress is tagging your address at their web based parser: [https://parserator.datamade.us/usaddress/](https://parserator.datamade.us/usaddress/)

But based on the sample address, it is tagging "# 12" as the OccupancyIdentifier, rather than identifying "#" as the OccupancyType and "12" as the OccupancyIdentifier

I may need to write up some special handling for # addresses and/or an issue should be raised with usaddress [https://github.com/datamade/usaddress/issues](https://github.com/datamade/usaddress/issues)

**Comment by fablet** [comment]
Appears to be resolved in #25
